rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -187,34 +187,36 @@ components:
       pattern: '^([1-9]([0-9]+)?)\.(0|([1-9]([0-9]+)?))(\.(0|([1-9]([0-9]+)?))(-.*)?)?$'
       example: 1.2, 2.4.3, 0.2.5-SNAPSHOT
     TransformRequest:
-      description: Transformation request.
       type: object
       required:
         - location
         - mimeType
-        - bytes
+        - metacardLocation
       properties:
         location:
-          type: string
           description: >
-            The URI of the product to transform. This URI must support HTTP GET at location to retrieve
-            the product and HTTP PUT with attachment at location/metadata/{metadataType} to store
-            metadata where {metadataType} is restricted to alphanumeric characters.
+            This URI must support HTTP GET to retrieve the file and HTTP PUT with attachment at
+            location/metadata/{metadataType} to store metadata, where {metadataType} is restricted
+            to alphanumeric characters.
+          type: string
+          format: uri
           minLength: 1
           maxLength: 2048
           example: https://www.example.com/mis/30f14c6c1fc85cba12bfd093aa8f90e3
         mimeType:
           type: string
-          description: The mime type of the ingested product.
+          description: The mime type of the file that can be retrieved at the location.
           minLength: 1
           maxLength: 255
           example: application/xml
-        bytes:
-          type: integer
-          format: int64
-          description: The size, in bytes, of the ingested product.
-          minimum: 1
-          example: 2147483648
+        metacardLocation:
+          description: >
+            This URI must support HTTP GET to retrieve the metacard xml.
+          type: string
+          format: uri
```

Comment:
❗Having the `format` field here adds the `@Valid` annotation in the generated `TransformRequest` class.  This causes issues, because no validator is configured.  Please remove this line.